src/Astreiko.Homework4/Program.cs

```diff
@@ -0,0 +1,375 @@
+                };
+            }
+        }
+
+        /// <summary>
+        /// Редактирование события
+        /// </summary>
+        private static void EditEvent()
+        {
+            var checkEdit = true;
+
+            while (checkEdit)
+            {
+                Console.Write("Input id for edit : ");
+
+                var searchId = Console.ReadLine().Trim();
+
+                var searchEvent = CheckEvent(searchId);
+
+                if (searchEvent.BoolResult)
+                {
+                    Console.WriteLine("Event found.");
+                    Console.WriteLine();
+
+                    Console.Write("Enter description : ");
+                    CurrentListEvents[searchEvent.NumberRow].Description = Console.ReadLine().Trim();
+
+                    var inputDate = CheckDate();
+                    CurrentListEvents[searchEvent.NumberRow].StartDate = inputDate.StartDate;
+
+                    CurrentListEvents[searchEvent.NumberRow].EndDate = inputDate.EndDate;
+
+                    Console.Write("Enter status : ");
+                    CurrentListEvents[searchEvent.NumberRow].Status = Console.ReadLine().Trim();
+
+                    Console.WriteLine("Event edited.");
+
+                    checkEdit = false;
+                }
+                else
+                {
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    Console.WriteLine("Event does not find. Please try again.");
+                    Console.WriteLine();
+                    Console.ResetColor();
+
+                    Console.Write("You want continious? (y - default or n)");
+
+                    if (Console.ReadLine().Trim() == "n") { checkEdit = false; }
+                };
+            }
+        }
+
+        /// <summary>
+        /// Прверка существования введенного идентификатора события
+        /// </summary>
+        /// <returns></returns>
+        private static (bool BoolResult, int NumberRow) CheckEvent(string searchId)
+        {
+            var vResB = false;
+            var vResI = 0;
+
+            for (int i = 0; i < CurrentListEvents.Count; i++)
+            {
+                if (CurrentListEvents[i].Id == searchId) { vResB = true; vResI = i; };
+            }
+
+            return (vResB, vResI);
+        }
+
+        /// <summary>
+        /// Создание события
+        /// </summary>
+        private static void CreateEvent()
+        {
+            Console.Write("Enter description : ");
+            var inputDesc = Console.ReadLine().Trim();
+
+            var inputDate = CheckDate();
+
+            Console.Write("Enter status : ");
+            var inputStatus = Console.ReadLine().Trim();
+
+            AddEventToList(inputDesc, inputDate.StartDate, inputDate.EndDate, inputStatus);
+        }
+
+        /// <summary>
+        /// Получение введенных дат
+        /// </summary>
+        /// <returns></returns>
+        private static (DateTime StartDate, DateTime EndDate) CheckDate()
```

Comment:
This tuple return type could be useful, but instead of simplicity and code reuse, code duplication is introduced